tests/conftest.py

```diff
@@ -22,7 +22,7 @@
 
 # Prevent calls to sleep
 def _sleep(*args):
-    raise Exception("Call to sleep")
+    pass
```

Comment:
This should also not be changed.

Reply:
I don't understand why it calls an exception. It's hard to rebuild a lot of older tests without looking for which methods are calling them.

Reply:
This is here to ensure that any call to `time.sleep()` within the test suite is an exception. Otherwise, the time it takes to run tests rapidly balloons due to many sleeps.

Reply:
Why not just return nothing, as I do?

Reply:
The exception actively encourages test-writers to use `mock.patch` to make sure `time.sleep()` is not called.

Reply:
An effort should be made to patch every integration test.